tests/archiving/test_import.py

```diff
@@ -64,6 +64,48 @@ def test_import_compressed(self):
         compare_obj = dircmp(path_original, path_import)
         self.assertEqual(len(compare_obj.diff_files), 0)
 
+    def test_unpack_to_nested_project(self):
+        pr = self.pr.open("nested")
+        pr_imp = pr.open("imported")
+        pr_imp.unpack(origin_path=self.arch_dir_comp, compress=True)
+
+        path_original = self.pr.path
+        path_import = pr_imp.path
+        path_original = getdir(path_original)
+        path_import = getdir(path_import)
+        print(f"DEBUG \n"
+              f" print(path_original, path_import)\n"
+              f"> {path_original, path_import}\n"
+              f" os.listdir(path_original)\n"
+              f"> {os.listdir(path_original)}\n"
+              f" os.listdir(path_import)\n"
+              f"> {os.listdir(path_import)}\n"
+              f"END DEBUG")
+        compare_obj = dircmp(path_original, path_import)
+
+        self.assertEqual(len(compare_obj.diff_files), 0)
+
+        pr.remove(enable=True)
+
+    def test_unpack_from_other_dir(self):
+        cwd = os.getcwd()
+        print("DEBUG")
+        print(f"cwd: {cwd}")
+        pack_path = os.path.join(cwd, 'exported')
+        print(f"pack_path: {pack_path}")
+        os.mkdir(path=pack_path)
+        pack_path_comp = os.path.join(pack_path, self.arch_dir_comp)
+        pack_path_csv = os.path.join(pack_path, 'export.csv')
+        print(f"pack_path_comp: {pack_path_comp}\n"
+              "pack_path_csv: {pack_path_csv}")
+        self.pr.pack(destination_path=pack_path_comp, csv_file_name=pack_path_csv, compress=False)
+        print(f"content of pack_path: {os.listdir(pack_path)}")
+        pr = self.pr.open("nested")
+        pr_imp = pr.open("imported")
+        pr_imp.unpack(origin_path=pack_path_comp, csv_file_name=pack_path_csv, compress=False)
+        compare_obj = dircmp(pack_path_comp, pr_imp.path)
+        self.assertEqual(len(compare_obj.diff_files), 0)
+
```

Comment:
Since we do not use the compressed file here, I would rather reflect this in the names. This confused me on reading.

```suggestion
    def test_unpack_from_other_dir(self):
        cwd = os.getcwd()
        print("DEBUG")
        print(f"cwd: {cwd}")
        pack_path = os.path.join(cwd, 'exported')
        print(f"pack_path: {pack_path}")
        os.mkdir(path=pack_path)
        pack_path_arch = os.path.join(pack_path, self.arch_dir)
        pack_path_csv = os.path.join(pack_path, 'export.csv')
        print(f"pack_path_arch: {pack_path_arch}\n"
              "pack_path_csv: {pack_path_csv}")
        self.pr.pack(destination_path=pack_path_arch, csv_file_name=pack_path_csv, compress=False)
        print(f"content of pack_path: {os.listdir(pack_path)}")
        pr = self.pr.open("nested")
        pr_imp = pr.open("imported")
        pr_imp.unpack(origin_path=pack_path_arch, csv_file_name=pack_path_csv, compress=False)
        compare_obj = dircmp(pack_path_arch, pr_imp.path)
        self.assertEqual(len(compare_obj.diff_files), 0)

```

Reply:
I have updated the name of the method to reflect this. I simply added `_uncompress` to the end.